src/color/shade.js

```diff
@@ -0,0 +1,33 @@
+// @flow
+
+import mix from './mix'
+
+/**
+ * Shades a color by mixing it with black. Leverages the mix color module.
+ *
+ * @example
+ * // Styles as object usage
+ * const styles = {
+ *   background: shade(0.25, '#00f')
+ * }
+ *
+ * // styled-components usage
+ * const div = styled.div`
+ *   background: ${shade(0.25, '#00f')};
+ * `
+ *
+ * // CSS in JS Output
+ *
+ * element {
+ *   background: "#00003f";
+ * }
+ */
+
+function shade(percentage: number, color: string) {
+  if (!percentage || !color) throw new Error('Passed insufficient arguments to shade, please pass a percentage and a color to be shaded.')
+  if (typeof percentage !== 'number' || percentage > 1) throw new Error('Passed an incorrect argument to shade, please pass a percentage less than or equal to 1.')
```

Comment:
I need to update this to capture the negative use case, but I'm not sure I like the idea of allowing either helper to do  the job of the other by accident (putting in a negative number). It feels like it would be better to limit it based on each helpers use case. Thoughts?